app/models/notification.rb

```diff
@@ -5,4 +5,7 @@ class Notification < ApplicationRecord
   scope :newest , -> {order  created_at: :desc}
   scope :check_send_to , -> (user){where("send_to_id = #{user}")}
   scope :check_send_from, -> (user){where("send_from_id  != #{user}")}
+  scope :admin_check_send_to_id, ->(current_admin){where "send_to_id = #{current_admin.id}"}
+  scope :admin_check_send_from_type, ->{where "send_from_type != 'Reported'"}
```

Comment:
{where "send_from_type != 'Reported'"}
=> {where.not send_from_type: 'Reported'}